src/app/common/KFilterBar.vue

```diff
@@ -507,16 +508,16 @@ function areFieldsSemanticallyIdentical(fieldsA: Fields, fieldB: Fields): boolea
   display: inline-flex;
   align-items: center;
   margin-left: auto;
-  padding-right: var(--spacing-xxs);
-  padding-left: var(--spacing-xxs);
+  padding-right: $kui-space-20;
+  padding-left: $kui-space-20;
   border-radius: 3px;
-  color: var(--grey-400);
+  color: $kui-color-text-neutral-weak;
 }
 
 .k-apply-suggestion-button:hover,
 .k-apply-suggestion-button:focus {
-  color: var(--white);
-  background-color: var(--blue-500);
+  color: $kui-color-text-inverse;
+  background-color: $kui-color-border-primary;
```

Comment:
issue: border color as a background color

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1314/commits/93a38ec687bb97965d618522e6447fa1efcb0e5e.